spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteTablePathsAction.java

```diff
@@ -423,8 +442,10 @@ public void testPositionDeletes() throws Exception {
         .hasSize(1);
   }
 
-  @Test
-  public void testPositionDeleteWithRow() throws Exception {
+  @TestTemplate
+  public void testV2PositionDeleteFilesWithRow() throws Exception {
+    assumeThat(formatVersion).isEqualTo(2);
```

Comment:
same as above, you need to pass the `formatVersion` to FileHelpers.writeDeleteFile(..) and let the test run with all format versions